aerosol_dynamics/particle_test.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from aerosol_dynamics import particle
+from . import ureg
```

Comment:
I have heard read we should we avoid any relative imports. So we may need to change this in the future. We also should leave `__init__.py` as empty as possible.

Reply:
I'll see what I can do about these imports.
__init__.py was necessary due to how pint works. Each file that imports pint creates a separate units registry, and the registries do not talk to each other. If we pass a radius from particle.py to particle_test.py and try to match it with a radius created in particle_test.py, pint will say the two radii have different units registries. As our file structure evolves with the project, that may be an issue we have to resolve.